src/components/Form.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import { useState } from 'react';
+
+type FormProps = {
+  onSubmit: (e: React.FormEvent<HTMLFormElement>) => void;
+};
+
+export default function Form({ onSubmit }: FormProps) {
+  const [inputValue, setInputValue] = useState('');
+
+  return (
+    <form onSubmit={onSubmit} className="flex flex-col gap-4 w-80">
+      <input
+        type="text"
+        placeholder="入力してください"
```

Comment:
[nits]
明文化した方がいいかな〜と思います。
```suggestion
        placeholder="フォロワー数を入力してください"
```